rust/linux-client/Cargo.toml

```diff
@@ -21,3 +21,4 @@ humantime = "2.1"
 resolv-conf = "0.7.0"
 thiserror = "1.0.57"
 tokio = { version = "1.36", default-features = false, features = ["rt", "macros", "signal"] }
+url = { version = "2.3.1", default-features = false }
```

Comment:
I'm not sure how it's deserializing URLs if I turned off all the features. I wonder if it's one of those glitches where this package doesn't ask for a feature, but another package in the workspace does, so we get a feature we didn't ask for and silently succeed when technically the Cargo.toml manifest is not correct.

Reply:
I'm not sure if that's consider a glitch, but I think it's that